payments-core/src/main/java/com/stripe/android/payments/core/authentication/threeds2/Stripe3ds2TransactionViewModel.kt

```diff
@@ -314,35 +323,56 @@ internal class Stripe3ds2TransactionViewModelFactory(
     @IOContext
     lateinit var workContext: CoroutineContext
 
+    /**
+     * Fallback call to initialize dependencies when injection is not available, this might happen
+     * when app process is killed by system and [WeakMapInjectorRegistry] is cleared.
+     */
+    override fun fallbackInitialize(arg: FallbackInitializeParam) {
+        DaggerStripe3ds2TransactionViewModelFactoryComponent.builder()
+            .context(arg.application)
+            .enableLogging(arg.enableLogging)
+            .workContext(Dispatchers.IO)
```

Comment:
since atm all our graphs provides `Dispatchers.IO` for `@IOContext` and `Dispatchers.Main` for `@UIContext`, we can create a dedicated `Module` class to initialize them. Will do this in a separate PR to reduce the size of this one